.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: node_js
+node_js: 4
+cache:
+  directories:
+  - node_modules
+sudo: false
+
+env:
+  - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+
+before_install:
+- openssl aes-256-cbc -K $encrypted_0cc7effbabbd_key -iv $encrypted_0cc7effbabbd_iv
+  -in super_secret.txt.enc -out super_secret.txt -d
```

Comment:
@DirtyF Be careful, it is on a new line so the key decryption is likely to fail. Line 18+19 should fit on one single line.